Rescale add int16 correctly

3l1 opened the pull request:
Differential Revision: D82906134

3l1 commented:
@pytorchbot label "release notes: none"

digantdesai reviewed (approved):
Review automatically exported from Phabricator review in Meta.

3l1 committed:
Rescale add int16 correctly (#14645)

Summary:



 we rescale to int32 using a different common scale(2*max(left scale,right scale))

the resccale(s) -> add[int32] -> rescale from TOSA is converted to a single add command stream instruction by Vela/Regor


bypass-github-export-checks
bypass-github-pytorch-ci-checks
bypass-github-executorch-ci-checks

Reviewed By: digantdesai, Ninja91

Differential Revision: D82906134
